src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -214,6 +215,7 @@ public static class ScriptConstants
         public static readonly long DefaultMaxRequestBodySize = 104857600;
 
         public static readonly ImmutableArray<string> SystemLogCategoryPrefixes = ImmutableArray.Create("Microsoft.Azure.WebJobs.", "Function.", "Worker.", "Host.");
+        public static readonly ImmutableArray<string> SystemLogCategoryPrefixesForV3 = ImmutableArray.Create("Host.Startup");
```

Comment:
Suggest changing the variable name to something like `RestrictedSystemLogCategoryPrefixes`